wiki/Guides/How_To_Get_Your_Map_Ranked/en.md

```diff
@@ -1,25 +1,19 @@
+Once your beaatmap has a decent amount of mods and approval, and you think it is ready to be ranked, you can get a [Beatmap Nominator](/wiki/People/Beatmap_Nominator). Check the team member's signature or userpage to see if they have a modding policy. If they are open to requests, ask them politely in-game if they would not mind taking a look at your map. Many of Beatmap Nominators are busy as there are an awful lot of pending maps to get through, so do not be too disheartened if you are turned down; however, having lots of mods and approval on your beatmap will help encourage Beatmap Nominators to look at your beatmap.
 
-## Step 3: Bubbles
+## Step 3: Nominations
 
-Just because your map is bubbled doesn't mean your work is over! Don't ignore any mods by normal users on your bubbled map — although only a BN or QAT can rank your map, anyone can still help improve it, and a better, more thoroughly modded map means an easier job for the BNG who eventually looks at it. Once you have applied a mod on your bubble the BN who originally gave the bubble will almost always be happy to come back and reinstate it. Keep in mind that you'll also have an easier time getting your bubble noticed by BNG members if it is high priority and on the first page of pending - giving your map some kudos stars to push it up in the list should make the wait a little shorter.
+Just because your beatmap got nominated does not mean you can sit back and relax! Do not ignore mods from normal users on your beatmap. Although only a Beatmap Nominator member can rank your map, anyone can improve it. A more thoroughly modded beatmap means an easier job for the Beatmap Nominator who looks at it. Once you have applied a mod on your nomination, the Beatmap Nominator who originally gave the nomination will be happy to come back and reinstate the nomination. Keep in mind that you will also have an easier time getting your nomination noticed by Beatmap Nominators if it is high priority. Having people build your hype train to push it up the priority list should make the wait for getting ranked shorter.
```

Comment:
```suggestion
Just because your beatmap got nominated does not mean you can sit back and relax! Do not ignore mods from normal users on your beatmap. Although only a Beatmap Nominator or Nomination Assessment Team member can rank your map, anyone can improve it. A more thoroughly modded beatmap means an easier job for the Beatmap Nominator who looks at it. Once you have applied a mod on your nomination, the Beatmap Nominator who originally gave the nomination will be happy to come back and reinstate the nomination. Keep in mind that you will also have an easier time getting your nomination noticed by Beatmap Nominators if it is high priority. Having people build your hype train to push it up the priority list should make the wait for getting ranked shorter.
```

It looks like the NAT can act like a BN in the modding and ranking process.
https://osu.ppy.sh/help/wiki/People/The_Team/Nomination_Assessment_Team